stories/setup/webpack.config.babel.js

```diff
@@ -48,6 +52,12 @@ module.exports = {
       /config$/,
       'core/client/config.js',
     ),
+
+    // This swaps the server side window object with a standard browser window.
+    new webpack.NormalModuleReplacementPlugin(
+      /core\/window/,
+      'core/browserWindow.js',
+    ),
```

Comment:
Why exactly don't we reuse the existing production config?

Reply:
I think we did this at one pt but I thought we were only going to pass what we needed. I can revisit this tho..